index.html

```diff
@@ -519,10 +520,10 @@ <h3>Proofs</h3>
         </pre>
 
         <p>
-The proof example above uses the <code>JcsSignature2020</code>
-<a>proof type</a> to produce a verifiable digital proof by canonicalizing the
-input data using the JSON Canonicalization Scheme [[RFC8785]] and then
-digitally signing it using an Ed25519 elliptic curve signature.
+The proof example above suggests an exemplary <code>json-signature-2022</code>
```

Comment:
```suggestion
The proof example above suggests a fictitious <code>json-signature-2022</code>
```

@TallTed what about "ficticious" instead? It's closer to what I was intending. We're trying to convey that "the thing above doesn't exist, we're only using it as an example that presumptively does X and Y." Thoughts?

Reply:
I'm OK with `fictitious`, tho optimally that fiction would carry through all layers, so the fictitious example looks as real as possible, while NOT being real (and having any descriptions that exist for any entities used also indicate that fictitiousness).

This is one of the reasons I fight to use reserved `example` domain names, phone numbers, etc., wherever they're meant to appear functional for the functionality we need while not carrying through to the physical world.

Reply:
What if we were to rename `json-signature-2022` to `example-signature-2022`? Would that be better?

Reply:
Much!

Reply:
Fixed in 0bcd83ce6237a68d217cd99441ec538d4efda5ca.